index.html

```diff
@@ -5132,6 +5132,12 @@ <h2>The <code>application/credential+ld+json</code> Media Type</h2>
     the use of a specific media type.
   </p>
 
+  <p>
+    This media type MUST NOT be used with <dfn>embedded proof</dfn>.
```

Comment:
```suggestion
    This media type MUST NOT be used to describe a Verifiable Credential with an <dfn>embedded proof</dfn>.
```

Reply:
@msporny @dlongley attempting to address your feedback.

Reply:
Well, this seems a bit inverted to me. I would expect, instead, that another media type (if we were to create it), e.g., `application/verifiable+credential+ld+json`, would contain a statement that says it MUST NOT be used with an unsecured (non-verifiable) credential. But doing this the other way around doesn't seem right.

It seems to me that a verifiable credential is still a credential -- and one could use either media type (the imagined one I just said or the one here) to describe it. But if you're seeing just `application/credential+ld+json` -- you should assume the sender meant to indicate that the credential should be treated as unsecured or that they didn't know / care if it was verifiable or not. Perhaps the language should be tweaked to say something more like that.

Now, if you receive a credential with such a media type and you want to determine if the credential has actually been secured in some way, you'll have to check its (optional) proof field (run verification software against it). But that should also be ok to do. This seems to be similar to receiving some JSON payload with `application/json` -- that may also be `application/something+json`. It's valid to do it, it's just less precise.

Reply:
As above, if I receive an `application/verifiable+credential+ld+json` document, and don't understand that media type, but do understand `application/credential+ld+json` or `application/ld+json` or `application/json`, can I get anything out of that document? If not (which I believe is the intention), then `application/verifiable+credential+ld+json` is not an appropriately constructed media type.

Also, if `application/verifiable+credential+ld+json` was a typo for `application/verifiable-credential+ld+json`, we have a strong argument against registering one, and maybe both, of these media types.

Reply:
@dlongley wrote:
> It's valid to do it, it's just less precise.

Yeah, this PR is raising questions around media type precision. The spec is going to have to say something about this now that we've re-opened the media types debate. The good news is that we have at least one logical way through that thicket now (where we didn't when this matter came up a few years ago). I've raised an issue #1015 to settle on some language around media type precision in the spec.

@TallTed wrote:
> can I get anything out of that document?

Yes, because there is a use case of publishing these things via schema.org in web pages and schema.org suggests publishing as `application/ld+json`, and people are going to follow that guidance and/or mess it up. When they inevitably mess it up (or do it on purpose), issue #1015 attempts to create language around deterministic steps that client and servers can perform to get us to where we need to go.

> `application/verifiable+credential+ld+json`

I think I'd be a -1 on a `+credential` structured suffix... again, it looks like the group is going down a path where we're going to create media type confusion based on the sheer number of media types we're considering.

Reply:
> [@OR13] Is `application/verifiable-credential+ld+json` a strict superset of `application/credential+ld+json`?
>
> Yes, and then by definition, they cannot be "equal".

I think the above is inverted, i.e., `application/verifiable-credential+ld+json` is a strict ***subset*** of `application/credential+ld+json`; that is, every `application/verifiable-credential+ld+json` is an `application/credential+ld+json`, but some `application/credential+ld+json` are not `application/verifiable-credential+ld+json`.

Also, I think they *could* be equal, ***IFF*** `application/credential+ld+json` were all made verifiable (i.e., if proofs were attached to all `application/credential+ld+json`), which will never happen, but that's not a definition issue.

Reply:
> which will never happen, but that's not a definition issue.

yeah, I agree with this.

I can also agree with your framing about JSON / RDF Inheritance...

The thing that a makes a `verifiable-credential` more than a `credential` is a `proof`.

We need to be clear about when `proof` is required and MUST be understood, and when it is optional and can be ignored safely.

Reply:
Resolving based on the merged suggestion here: https://github.com/w3c/vc-data-model/pull/1014/commits/a99fe01b917e5ce160cf78031e2d6e12546d6630